lms/djangoapps/mobile_api/users/views.py

```diff
@@ -221,6 +221,7 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
           * video_outline: The URI to get the list of all vides the user can
             access in the course.
           * id: The unique ID of the course.
+          * channel_id: A unique "clean" (alphanumeric with '_') ID of the course.
```

Comment:
Updated to using subscription_id.  I wasn't planning to update mobile3 until the review was complete (to avoid requiring you guys to keep updating your code), unless there's an app-required reason to do so.